runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixPlugin.java

```diff
@@ -47,7 +54,14 @@
 )
 public class TearsOfGuthixPlugin extends Plugin
 {
-	private static final int TOG_REGION = 12948;
+	static final int TOG_REGION = 12948;
+	private static final Set<Integer> TEARS_OBJECTS = new ImmutableSet.Builder<Integer>().

```

Comment:
Use [`ImmutableSet.of(E... elements)`](https://guava.dev/releases/19.0/api/docs/com/google/common/collect/ImmutableSet.html#of(E,%20E,%20E,%20E,%20E,%20E,%20E...)) instead of a builder.